freezeyt/cli.py

```diff
@@ -38,9 +39,18 @@
               default=None,
               help='Stop on the first error')
 def main(
-    app, dest_path, output, prefix,
-    extra_pages, config_file, config_var, progress, cleanup, gh_pages, fail_fast
-):
+    app: str,
+    dest_path: str,
+    output: Optional[str],
+    prefix: Optional[str],
+    extra_pages: Optional[List[str]],
+    config_file: Optional[TextIO],
```

Comment:
How do we know that is `TextIO` instead for example `BinaryIO`?

Reply:
It's the default for [`click.File`](https://click.palletsprojects.com/en/8.1.x/api/#click.File)